recipes/pyrser/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "pyrser" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/py2.py3/p/{{ name }}/{{ name }}-{{ version }}-py2.py3-none-any.whl
```

Comment:
Installing from wheels is really not a great way to do this. Is there any way to get an sdist/release tarball from upstream?

Reply:
There are no current release tars for this repository, but I submitted an issue for this repo to add one.
 I used this previous PR as a reference https://github.com/conda-forge/tensorflow-metadata-feedstock/blob/master/recipe/meta.yaml

Reply:
The only way to do it currently is via a wheel (there are no tarballs). We can make an issue on the repo and include a comment in the recipe that we will change if possible. A recent PR from a teammate was just merged, with a wheel, at the encouragement of a staged-recipes maintainer: https://github.com/conda-forge/staged-recipes/pull/14610#issuecomment-831491197